src/DotNet.Status.Web/Controllers/AlertHookController.cs

```diff
@@ -181,6 +181,8 @@ private NewIssue GenerateNewIssue(GrafanaNotification notification)
 {string.Format(BodyLabelTextFormat, GetUniqueIdentifier(notification))}
 
 </details>
+
+{(options.NotificationTarget == "@dotnet/dnceng" ? "### Release Note Description\nDo not include in release notes. (Note to devs: change this if this alert results in a code change.)" : "")}
```

Comment:
This doesn't feel like a great thing to hard code into the middle of the controller, especially in the middle of a formatted string.  AzDO issues are going to be the same way too.  Can we use some label or something and have the build validation ignore those labels?  That seems a bit better than spreading all this logic all over our services.

Reply:
Something like the build validation could ignore anything with "Grafana Alert" or "External Linked Issue" or something (rather than forcing all those issues to have this confusing text in them in all the places they are generated).